ArduCopter/mode_poshold.cpp

```diff
@@ -41,8 +41,7 @@ bool ModePosHold::init(bool ignore_checks)
     pilot_roll = 0.0f;
     pilot_pitch = 0.0f;
 
-    // compute brake_gain
```

Comment:
It's not clear what's changed here.. but let's not remove the comment.

Reply:
Comment is back.